pkgs/by-name/co/copilot-language-server/package.nix

```
@@ -24,30 +25,46 @@ let
     }
     ."${stdenvNoCC.hostPlatform.system}"
       or (throw "Unsupported system: ${stdenvNoCC.hostPlatform.system}");
-in
 
+  executableName = "copilot-language-server";
+  fhs =
+    { copilot-language-server }:
+    buildFHSEnv {
+      name = executableName;
```

```suggestion
      name = executableName;
```

I updated this part in the patch, why not including it?